datapackage_pipelines_assembler/processors/dump_to_s3.py

```diff
@@ -20,6 +20,17 @@
                  {'name': 'owner', 'type': 'string'},
                  {'name': 'ownerid', 'type': 'string'},
                  {'name': 'findability', 'type': 'string'},
+                 {'name': 'stats', 'type': 'object', 'properties': {
```

Comment:
Almost :)
This should use the 'es:schema' property (and not properties), and inside it should be a regular table schema.
See line 18 ^ for an example